### DISCUSSION TITLE
Recover Account no Passkey no recovery Codes

### DISCUSSION THREAD

**Discussion by nocluewhatido** [ente-io/ente#3526] [Q&A]
Hello everyone,

Unfortunately I fucked up big times. I just changed the OS of my phone from Android to GraphenOS. 
I didn't realize that I must have activated a passkey for Ente Auth on my phone and now I can't log in. I just have my Password. 

The big bummer is that I have the backup codes stored in my Proton Drive Cloud. However, I need the authenticator code to log in to Proton Drive. And, of course, I'm an idiot and have also saved the Proton Drive backup codes in the Proton Drive Cloud.

Is there anything I can do to recover my accounts?

God help me I have all my photos and documents stored in the cloud. I also wanted to save it to my local hard drive as a backup but I haven't done that yet.

**Comment by Damariobros** [comment]
Email support@ente.io from the email used to create your Ente account and explain the situation. They will be able to remove the passkeys from your account and disable passkeys, and then you can use your email and password to sign in and get access to all your accounts back.

I would recommend multiple backups of your recovery key and backup codes in different locations, for example writing them down and putting them in a safe, storage in a usb drive, storage in a cloud not accessed with Ente Auth.

Seems chicken-and-the-egg situations are pretty common, be careful out there 😅

**Comment by taivlam** [comment]
If you have your Ente account recovery key, then try to use that first to obtain access to your account - as described in this FAQ [page](https://ente.io/faq/security-and-privacy/forgot-password/).

As a basic approach, I would write down your recovery key.  Personally, I save this in my password manager's entry for my Ente account.

For the future, it's best to save the TOTP 2FA for Ente Auth either in a password manager (which can save TOTP codes, 6-digit codes that change every 30 seconds) or in another dedicated TOTP app.  ([Aegis](https://getaegis.app/) is a suggestion, if you are on Android.)  This would be another method of 2FA, in case you ever loose all passkeys.
